### PR TITLE
Updated README.md 

### PR DESCRIPTION
**Pull request by kev1n80**
Updated the README.md file to descripe the repo.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,37 @@
...
+      
+      <h3>About Me</h3>
+      <p> Click here to get a random fact about me</p>
+      <button onclick="addRandomFact()">Random Facts About Me</button>
+      <div id="fact-container"></div>
+      <p>I grew up in Miami, Florida surrounded by a Latinx community and playing video games such as Starcraft 2 and Call of Duty. 
+          I am currently a rising Sophomore at Northeastern University studying computer science. 
+          During my time at Northeastern I joined a salsa/bachata dance club called Kaliente in order to maintain some resemblace of home while I was away studying in Boston. 
+          Growing up in Miami, a city centered around cars, I am fascinated with the urban-planning model called superblocks that are being implemented in Spain! 
+          I hope to one day travel to Spain and experience their culture rich cities.
```
Comment (saucepwn) [inline portfolio/src/main/webapp/index.html:27]: Try to wrap all lines to either 80 or 100 characters so they can fit within a standard editor window

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,16 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact() {
+  const facts =
+      ['I love to dance!', 'I am a lefty', 'I use to fence foil', 'I am Peruvian!', 'I reached Plat in SC2', 'I have never been outside of the Americas'];
```
Comment (saucepwn) [inline portfolio/src/main/webapp/script.js:32]: I'm a lefty too!

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -7,3 +7,12 @@
...
 }
+
+#fact-container {
+  margin-top: 20px;
+}
+
+#portrait {
+    height:  100px;
+    width: auto;
+}
```
Comment (saucepwn) [inline portfolio/src/main/webapp/style.css]: Add newline at the end of the file

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -7,3 +7,12 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#fact-container {
+  margin-top: 20px;
+}
+
+#portrait {
+    height:  100px;
```
Comment (saucepwn) [inline portfolio/src/main/webapp/style.css]: formatting: remove extra space

**Review by saucepwn — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (kev1n80) [reply portfolio/src/main/webapp/script.js:32]: hahaha lefty squad!

**Pull request merged**